app/views/layouts/_google_analytics.html.erb

```diff
@@ -7,5 +7,79 @@ m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
 
 ga('create', '<%= Settings::GoogleAnalytics.google_analytics_code %>', 'auto');
 ga('send', 'pageview');
+
+$('[data-ga-category]').on('click', function() {
+  var category = $(this).data('ga-category');
+  var action = $(this).data('ga-action');
+  var label = $(this).data('ga-label');
+  ga('send', {
+    hitType: 'event',
+    eventCategory: category,
+    eventAction: action,
+    eventLabel: label
+  });
+});
+
+window.addEventListener('beforeunload', (event) => {
+  if((window.location.href.indexOf("signup") > -1) || (window.location.href.indexOf("profile") > -1)) {
```

Comment:
Same as my other comment...

`window.location.href.indexOf("profile") > -1` will evaluate to true for:
1. the new flow profile `/i/profile`
2. the old flow profile `/profile`
3. **and** the old flow "signup profile" step `/signup/profile`

Wouldn't we want to differentiate between the three of them?

Reply:
This is capturing the new flow, so I updated the index check.